mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule Postgrex.Mixfile do
   def project do
     [app: :postgrex,
      version: "0.9.2-dev",
-     elixir: "~> 1.0",
+     elixir: "~> 1.1",
```

Comment:
And make it clear the docs. "Expects a pid for Elixir 1.0. A pid or a name can be given from Elixir 1.1."